### PR TITLE
Feature/update double input

### PR DESCRIPTION
**Pull request by Dragonesis**

### CODE REVIEW

File: src/components/DoubleInput/DoubleInput.js [hunk 1]
```diff
@@ -19,80 +18,68 @@ type Props = {|
...
+  +onChangeRight: ?((value: string) => void),
   +onClose: ?(() => void),
-  +valueFiat: string,
-  +valueAmount: string,
-  +errorMessageAmount: string,
+  +valueLeft: string,
+  +valueRight: string,
+  +placeholderLeft: string,
+  +placeholderRight: string,
+  +errorMessage: string,
```
Comment (stcherenkov) [inline src/components/DoubleInput/DoubleInput.js]: `errorMessage` is always for both inputs?

File: src/components/DoubleInput/DoubleInput.js [hunk 2]
```diff
@@ -19,80 +18,68 @@ type Props = {|
...
   +isLoading: boolean,
```
Comment (stcherenkov) [inline src/components/DoubleInput/DoubleInput.js]: Loading can happen for both inputs only?

File: src/components/DoubleInput/DoubleInput.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
 // @flow
 
-import React, { PureComponent, Fragment } from 'react'
-import classNames from 'classnames'
-import { JInput, JText, JLoader, JIcon } from 'components/base'
+import React, { PureComponent } from 'react'
+import { JInput, JLoader, JIcon } from 'components/base'
 
 /*
```
Comment (flexxnn) [inline src/components/DoubleInput/DoubleInput.js]: remove it

File: src/components/DoubleInput/DoubleInput.js [hunk 4]
```diff
@@ -19,80 +18,68 @@ type Props = {|
...
+  +errorMessage: string,
   +isLoading: boolean,
-  +isActive: boolean,
 |}
 
 class DoubleInput extends PureComponent<Props> {
   static defaultProps = {
     isLoading: false,
-    isActive: false,
     onClose: null,
```
Comment (flexxnn) [inline src/components/DoubleInput/DoubleInput.js]: add here
`onChangeRight: null`

File: src/components/DoubleInput/DoubleInput.js [hunk 5]
```diff
@@ -19,80 +18,68 @@ type Props = {|
...
-                    <JLoader color='blue' />
-                  </div>
-                )}
-              </Fragment>
-            )}
-          </div>
+          )}
         </div>
-        {onClose &&
+        {!onClose &&
```
Comment (flexxnn) [inline src/components/DoubleInput/DoubleInput.js]: close should only appears when onClose is exists in the props

File: src/components/DoubleInput/DoubleInput.js (hunk 1 above)
Comment (flexxnn) [reply src/components/DoubleInput/DoubleInput.js]: Yes, and I think we should add the infoMessage (yellow message) here. We need when gasPrice value is too small

File: src/components/DoubleInput/DoubleInput.js (hunk 2 above)
Comment (flexxnn) [reply src/components/DoubleInput/DoubleInput.js]: In design we have only one loader, I think in send asset form thereis only one place with this loader (when fiat conversion rate is loading)

File: src/components/DoubleInput/DoubleInput.js [hunk 6]
```diff
@@ -28,45 +32,63 @@ class DoubleInput extends PureComponent<Props> {
...
+        {onClose &&
         <div className='close' onClick={onClose}>
           <JIcon name='padding-cross' size='medium' color='gray' />
         </div>}
+        {errorMessage &&
+        <div className='message'>
+          <JText value={errorMessage} color='red' size='small' />
+        </div>
+        }
+        {warningMessage &&
```
Comment (flexxnn) [inline src/components/DoubleInput/DoubleInput.js]: I think, error has biggest priority than warning, when we have both messages

`warningMessage && !errorMessage `

**Pull request merged**